environments/published.go

```diff
@@ -74,6 +74,8 @@ var PublishedApis = map[string]ApiAppId{
 	"MileIqDashboard":                   "f7069a8d-9edc-4300-b365-ae53c9627fc4",
 	"MileIqRestService":                 "b692184e-b47f-4706-b352-84b288d2d9ee",
 	"MixedReality":                      "c7ddd9b4-5172-4e28-bd29-1e0792947d18",
+	"Microsoft.Azure.Frontdoor"          "ad0e1c7e-6d38-4ba4-9efd-0bc77ba9f037",
+	"Microsoft.AzureFrontDoor-Cdn"       "205478c0-bd83-4e1b-a9d6-db63a3e1e1c8",
```

Comment:
Could these be pascal-case without punctuation like the other entries?

Reply:
Changed. But don't you think that it's easier for users to use official application names instead of looking how its hardcoded in the provider even though this is an unofficial source?

Reply:
Thanks. Unfortunately there's no general convention across apps / APIs so I elected to normalize the names so make them slightly easier to discover. Map keys without punctuation are also easier to re-use across DSLs, for example if you wanted to represent this in YAML or HCL.

Reply:
Thanks for explanation.